include/swift/Serialization/ModuleFormat.h

```diff
@@ -379,6 +379,7 @@ enum class DefaultArgumentKind : uint8_t {
   NilLiteral,
   EmptyArray,
   EmptyDictionary,
+  StoredProperty,
```

Comment:
Since you changed the module format please also increment the minor version number at the top of this file